src/GovUk.Education.ExploreEducationStatistics.Data.Services/TableBuilderService.cs

```diff
@@ -110,6 +116,33 @@ public async Task<Either<ActionResult, TableBuilderResultViewModel>> Query(
                 });
         }
 
+        public async Task<Either<ActionResult, Dictionary<string, List<LocationAttributeViewModel>>>> Query(
+            Guid releaseVersionId,
+            FullTableQuery query,
+            long boundaryLevelId,
+            CancellationToken cancellationToken = default)
```

Comment:
I'd be inclined to rename this method to distinguish it from the other `Query` method - maybe `QueryForBoundaryLevel` - to help people who spot the methods are named the same but not catch that the return types are different.